sections/main-product.liquid

```diff
@@ -1,729 +1,96 @@
-<section
-  id="MainProduct-{{ section.id }}"
-  class="page-width section-{{ section.id }}-padding"
-  data-section="{{ section.id }}"
->
-  {{ 'section-main-product.css' | asset_url | stylesheet_tag }}
-  {{ 'component-accordion.css' | asset_url | stylesheet_tag }}
-  {{ 'component-price.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rte.css' | asset_url | stylesheet_tag }}
-  {{ 'component-slider.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rating.css' | asset_url | stylesheet_tag }}
-  {{ 'component-loading-overlay.css' | asset_url | stylesheet_tag }}
-  {{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+<div class="color-{{ section.settings.color_scheme }}">
```

Comment:
I wonder if drawers triggered from the product information should match the color scheme of it 🤔 [video](https://screenshot.click/12-21-rwzj8-swm88.mp4)

But I think we're wanting to have a specific general color setting for drawers in general in the future. What do you think @Oliviammarcello ?

Reply:
Since merchants have the ability to change the drawer colour I wonder if that is good enough. I could see use cases where merchants would want the two to be different since it would apply to all drawers in the theme (cart, filters, etc.)

Reply:
Yeah Meli had created an issue for it here: https://github.com/Shopify/dawn/issues/1403